invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides

```diff
@@ -430,7 +430,9 @@ dl.details-list {
   box-shadow: none !important;
 }
 
-
-.contents-container {
-  display: contents !important;
+.auto-column-grid {
```

Comment:
I would also appreciate understanding (comment? other?) when this should be used.